src/services/project-type-specifics.test.js

```diff
@@ -0,0 +1,18 @@
+import { getDocumentationLink } from './project-type-specifics'
+
+import type { ProjectType } from '../types';
+
+describe('getDocumentationLink', () => {
+  it("should get the documentation link for React", () => {
+    expect(getDocumentationLink('create-react-app')).toEqual('https://github.com/facebook/create-react-app#user-guide')
+  })
+
+  it('should get the documentation link for Gatsby', () => {
+    expect(getDocumentationLink('gatsby')).toEqual('https://www.gatsbyjs.org/docs/')
+  })
```

Comment:
I wonder if these tests are too specific...

What I mean by that is, the `getDocumentationLink` function doesn't do any work, it just returns a string, and by checking that exact string, the tests will fail when we update the documentation link. I can't think of a lot of reasons that the failure would be genuine (since there's no real room for someone to make an error inside the method).

I like the final clause, checking that it throws. But maybe we can just check that both values are strings, and they aren't equal?

```
it("provides strings for Gatsby and CRA", () => {
  const gatsbyString = getDocumentationLink('gatsby');
  const createReactAppString = getDocumentationLink('create-react-app');

  expect(typeof gatsbyString).toEqual('string');
  expect(typeof createReactAppString).toEqual('string');

  expect(gatsbyString).not.toBe(createReactAppString);
}
```

I know some feel like it's bad form to have multiple `expect`s per test. I'm happy either way: you could create 3 `it` statements for these checks if you prefer!

Reply:
> I wonder if these tests are too specific...
> 
> What I mean by that is, the getDocumentationLink function doesn't do any work, it just returns a string, and by checking that exact string, the tests will fail when we update the documentation link. I can't think of a lot of reasons that the failure would be genuine (since there's no real room for someone to make an error inside the method).
>
> I like the final clause, checking that it throws. But maybe we can just check that both values are strings, and they aren't equal?

I think you're right, there is a level of too specific/too fine-grained with tests where the test becomes unhelpful and counter-productive, and I think this test is passed that line. This is fixed up now!